Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
```

Comment:
very good point for `rspec`, but first, for the same reasons as `ci_reporter` I think `coveralls`'s place is in the to Gemfile. Nothing depends on `coveralls` other than the top level package.

As for `rspec`, I do think it belongs in `logstash-devutils` since to run specs in that gem, you need rspec. So in that respect, it is probably redundant to also have it in the top Gemfile. I'll see if we can clean that, but I don't think we should delay merging just for that, but I'll try to see if there is a problem in just leaving it in `logstash-devutils`.